library/core/src/ptr/alignment.rs

```diff
@@ -193,16 +311,17 @@ impl hash::Hash for Alignment {
     }
 }
 
-#[cfg(target_pointer_width = "16")]
```

Comment:
I was specifically asked to do this this way -- https://github.com/rust-lang/rust/pull/95361#discussion_r846530028 -- so unless you feel particularly strongly, I'd say to just leave it like this.

(By having `AlignmentEnum16` not cfg'd out it means you know it actually compiles without checking on a 16-bit target.)

Reply:
I'm fine reverting this change tbqh